Dockerfile

```diff
@@ -55,6 +55,7 @@ ENV ILITOOLS_CONFIG_DIR=/config
 ENV ILITOOLS_CATALOGUES_DIR=/catalogues
 ENV ILITOOLS_MODELS_DIR=/models
 ENV ILITOOLS_PLUGINS_DIR=/plugins
+ENV ILITOOLS_CACHE_DIR=/cache
```

Comment:
PP: ILI_CACHE im `docker-entrypoint.sh` direkt mit dem `ILITOOLS_CACHE_DIR` überschreiben. So müssen wir nicht die Dokumentation von ilitools zitieren / wiederhohlen.
REQ: Funktion in Readme.md dokumentieren.